src/test/scala/mesosphere/marathon/core/matcher/base/util/ActorOfferMatcherTest.scala

```diff
@@ -28,12 +68,36 @@ class ActorOfferMatcherTest extends AkkaUnitTest {
         })
         val offer = MarathonTestHelper.makeBasicOffer().build()
 
-        val offerMatcher = new ActorOfferMatcher(probe.ref, None)(scheduler)
+        val offerMatcher = new ActorOfferMatcher(probe.ref, None, new Provider[MarathonScheduler] {
+          override def get() = mock[MarathonScheduler]
+        })(scheduler)
         val offerMatch: MatchedInstanceOps = offerMatcher.matchOffer(offer).futureValue
 
         offerMatch.offerId should not be (offer.getId)
         offerMatch.offerId.getValue should be("other")
       }
     }
+
+    def offerWithFaultRegion(faultRegion: String) = {
+      Offer.newBuilder()
```

Comment:
I really wish we would move to a Scala native protobuf package with case classes to get rig of these builders. Anyhow, thanks for the factory method :).

In the passed we would follow the pattern of a `class Fixture` which would hold this method and then create the fixture in each test. What do you think?

Reply:
by scala protobuf you mean this one? https://github.com/scalapb/ScalaPB we actually did not like about it that it requires python installed on your computer...

Reply:
regarding the fixture - will do :)

Reply:
Oh, yeah. That is a minus. Is there no alternative around? @timcharper, you basically know every Scala package right? Do you know an alternative?